<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeInputWidget</class>
 <widget class="QWidget" name="TimeInputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="absoluteTab">
      <attribute name="title">
       <string>Absolute</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QCalendarWidget" name="calendarWidget">
         <property name="gridVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>274</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>Time:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="relativeTab">
      <attribute name="title">
       <string>Relative</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSpinBox" name="numberBox">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Days</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Weeks</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Years</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ago</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>213</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>378</width>
           <height>268</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
